src/planscape/collaboration/services.py

```diff
@@ -34,13 +34,12 @@ def get_role(user, instance):
         return "Creator"
     else:
         content_type = ContentType.objects.get_for_model(instance)
-        return (
-            UserObjectRole.objects.filter(
-                collaborator=user, content_type=content_type, object_pk=instance.pk
-            )
-            .first()
-            .role
+        roles = UserObjectRole.objects.filter(
+            collaborator=user, content_type=content_type, object_pk=instance.pk
         )
+        if not roles:
```

Comment:
I think this is an edge case we didn't encounter before, b/c we were always prefiltering our PlanningArea for records with `get_for_user()` which requires the user has some permission before getting to this point.